### PR TITLE
Doc: Document caching benefits, behavior, and configuration

### LINKED ISSUE #208: Document caching behavior, benefits, and configuration

### PR DESCRIPTION
**Pull request by nicksay**
Closes #208

### CODE REVIEW

**Comment by nicksay** [comment]
Currently staged at https://nicksay.github.io/spfjs/documentation/caching/

File: doc/documentation/caching.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## When the cache is used
+
+Before making a request, SPF checks the local cache to see if a
+valid cached response is available for the requested URL.  If
+one is found, then it will be used instead of sending a request
+over the network.  If not, after the response is received, SPF
+will place that response in the local cache for future use.
+
+By default, SPF uses a caching model that matches the back-
```
Comment (awbraunstein) [inline doc/documentation/caching.md]: should back-forward be one word? It renders as back- forward.

File: doc/documentation/caching.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+`cache-unified` config value is set to `true`, then the
+prefetched response will be available for all navigation like
+other cached responses.
+
+
+## Configuring the cache
+
+You may configure the cache using parameters to control the
+total number and lifetime of entries in the cache.  Increasing
+these settings will increase the chance of finding a valid
```
Comment (awbraunstein) [inline doc/documentation/caching.md]: valid cached -> valid, cached

I think you should have a comma here. Not really an expert though.

File: doc/documentation/caching.md (hunk 1 above)
Comment (nicksay) [inline doc/documentation/caching.md]: Yes, thanks; done.

File: doc/documentation/caching.md (hunk 2 above)
Comment (nicksay) [inline doc/documentation/caching.md]: I think this is an optional comma case, but added for clarity; done.

**Comment by awbraunstein** [comment]
lgtm

**Pull request merged**